### PR TITLE
Update build.gradle quickBuild task (DEV)

### PR DESCRIPTION
**Pull request by SamuraiKek**
Changed ktlintDeviceReleaseCheck to ktlintDeviceReleaseSourceSetCheck to prevent quickBuild from failing locally.

Test by running ./gradlew quickBuild

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -73,7 +73,7 @@ task quickBuild {
     dependsOn ':Corona-Warn-App:assembleDeviceRelease'
     dependsOn ':Corona-Warn-App:testDeviceReleaseUnitTest'
     dependsOn ':Corona-Warn-App:lintDeviceRelease'
-    dependsOn ':Corona-Warn-App:ktlintDeviceReleaseCheck'
+    dependsOn ':Corona-Warn-App:ktlintDeviceReleaseSourceSetCheck'
```
Comment (mtwalli) [inline build.gradle:76]: can you please change also in `quickBuildWithFixes` task ?

Comment (harambasicluka) [reply build.gradle:76]: we could even think about removing the quick from the name :D
image

**Review by harambasicluka — DISMISSED** [review]

File: build.gradle (hunk 1 above)
Comment (SamuraiKek) [reply build.gradle:76]: > we could even think about removing the quick from the name :D
You're lucky, it takes me 28m on average...

Comment (SamuraiKek) [reply build.gradle:76]: > can you please change also in `quickBuildWithFixes` task ?

Done

**Comment by d4rken** [comment]
> Changed ktlintDeviceReleaseCheck to ktlintDeviceReleaseSourceSetCheck to prevent quickBuild from failing locally.

Why does one fail and the other one doesn't and why does this happen now and not before? :thinking:

**Comment by SamuraiKek** [comment]
> > Changed ktlintDeviceReleaseCheck to ktlintDeviceReleaseSourceSetCheck to prevent quickBuild from failing locally.
> 
> Why does one fail and the other one doesn't and why does this happen now and not before? 🤔

From my discussion with @mtwalli yesterday, I think the name changed.

**Review by mtwalli — APPROVED** [review]
Lgtm

**Comment by mtwalli** [comment]
> > > Changed ktlintDeviceReleaseCheck to ktlintDeviceReleaseSourceSetCheck to prevent quickBuild from failing locally.
> > 
> > 
> > Why does one fail and the other one doesn't and why does this happen now and not before? 🤔
> 
> From my discussion with @mtwalli yesterday, I think the name changed.

We upgraded ktlint two times, apparently the task name changed in the meantime.

> why does this happen now and not before?
I don't think that anyone was using these tasks recently, I never used it. As Luka already mentioned it is not really quick and we should remove it

**Review by harambasicluka — APPROVED** [review]

**Pull request merged**